src/jobs/newsletters.ts

```diff
@@ -27,35 +27,38 @@ const isTrue = (str: string | undefined): boolean =>
 
 const rowToNewsletter = ({
 	[THEME_INDEX]: theme,
+	// column 2 ("name") is not used.
 	[GROUP_INDEX]: group,
-	3: name,
-	4: frequency,
-	5: description,
-	6: identityName,
-	7: listIdV1,
-	8: listId,
-	9: brazeSubscribeEventNamePrefix,
-	10: brazeNewsletterName,
-	11: brazeSubscribeAttributeName,
-	12: brazeSubscribeAttributeNameAlternate,
-	13: campaignName,
-	14: campaignCode,
-	15: restricted,
-	16: paused,
-	17: emailConfirmation,
+	3: cancelled,
```

Comment:
What do you think about placing this column near the other "status" columns (`restricted` at index 16 and `paused` at index 17)?

Reply:
I would tend to put it nearer the start of the row as it's and a key piece of information that in theory should only change (at most) once in the life of a newsletter - but not a strong opinion. Personally, if there are 4 boolean fields next to each other I'm more likely to get them mixed up.
Lets ask the people who use the spreadsheet what they would prefer overall.